BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -56,15 +69,12 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
         let item = UIBarButtonItem(image: UIImage(systemName: "gearshape.fill"), style: UIBarButtonItem.Style.plain, target: self, action: #selector(self.settingsButtonPressed))
         self.navigationItem.rightBarButtonItem = item
         
-        self.view.addSubview(self.lastUpdatedView)
+        stackView.addArrangedSubview(self.lastUpdatedView)
```

Comment:
How about configuring each individual view up here and then afterwards, toward the bottom of this method, setting the arranged subviews all at once? I think it would increase readability by decreasing the scope needed to see which views are in the stackView and in which order.

Reply:
Open to it, seems like a candidate for a follow up change.